andela/bp-esa-frontend
# 160437219(sub-task) Implement report page with mock data

#### What does this PR do?
- Add SCSS style loader to webpack config
- Move images to public folder
- Remove files that are no longer needed
- Restructure the app routing
- Add the Report Page Component
#### Description of Task to be completed?
Create the reporting page powered with mock data
#### How should this be manually tested?
- Clone and set up the project using the instruction in the README.
- Use`$ npm start` to start the application
- Visit `http://localhost:3000/report` to see the report page.
- **NOTE**: The port that the application starts might change on your local machine
#### Any background context you want to provide?
- I had to remove a lot of files that are no longer needed in the codebase and that was what led to the over 30 file changes in this pull request.
#### What are the relevant pivotal tracker stories?
This pull request is for a task under the story #160437219
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-20 at 3 41 06 pm" src="https://user-images.githubusercontent.com/26048536/45826088-9cdf0c80-bceb-11e8-9767-3a492364aa8a.png">
<img width="1440" alt="screen shot 2018-09-20 at 3 41 23 pm" src="https://user-images.githubusercontent.com/26048536/45826111-a9636500-bceb-11e8-8e3e-59aeb1803899.png">
